components/match2/wikis/rocketleague/match_group_input_custom.lua

```diff
@@ -186,8 +186,8 @@ function matchFunctions.getExtraData(match)
 		showh2h = showh2h,
 		isfeatured = matchFunctions.isFeatured(match),
 		casters = MatchGroupInput.readCasters(match),
-		hasopponent1 = Logic.isNotEmpty(opponent1.name) and opponent1.type ~= Opponent.literal,
-		hasopponent2 = Logic.isNotEmpty(opponent2.name) and opponent2.type ~= Opponent.literal,
+		hasopponent1 = opponent1.name ~= nil and opponent1.name:match('[^ /]') ~= nil and opponent1.type ~= Opponent.literal,
```

Comment:
untested, but more independent of the name building (which technically could get changed in the future):
```lua
		hasopponent1 = matchFunctions._checkForNonEmptyOpponent(opponent1),
		hasopponent2 = matchFunctions._checkForNonEmptyOpponent(opponent2),
```

```lua
function matchFunctions._checkForNonEmptyOpponent(opponent)
	return opponent.type ~= Opponent.literal and (
		opponent.type == Opponent.team and Logic.isNotEmpty(opponent.template) or
		Opponent.typeIsParty(opponent.type) and
			Array.any(opponent.match2players, function(player) return Logic.isNotEmpty(player.name) end)
	)
end
```
or alternatively
```lua
function matchFunctions._checkForNonEmptyOpponent(opponent)
	if Opponent.typeIsParty(opponent.type) then
		return Array.any(opponent.match2players, function(player) return Logic.isNotEmpty(player.name) end)
	elseif opponent.type == Opponent.team then
		return Logic.isNotEmpty(opponent.template)
	end

	--literal case
	return false
end
```